hkmc2/shared/src/main/scala/hkmc2/semantics/Resolver.scala

```diff
@@ -314,8 +323,24 @@ class Resolver(tl: TraceLogger)
     defn match
     
     // Case: instance definition. Add the instance to the context.
-    case defn @ TermDefinition(k = Ins, sym = sym, sign = sign) =>
+    case defn @ TermDefinition(_, Ins, sym, pss, tps, sign, body, _, TermDefFlags(isMethod), modulefulness, annotations) =>
       log(s"Resolving instance definition ${defn.showDbg}")
+      
+      pss.foreach(_.params.foreach(resolveParam(_)))
```

Comment:
Why does everyone always forget about the `restParam`? 😫

Reply:
😣